### PR TITLE
Deployment instructions

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +66,129 @@ npm run lint
...
+## How to deploy
+
+### Create your application on App Engine
+
+- Create a [project on GCP](https://console.cloud.google.com/projectcreate).
+
+- Make sure that the APIs listed below are **Enabled**:
+  - [Cloud Resource Manager API](https://console.cloud.google.com/apis/library/cloudresourcemanager.googleapis.com), 
+  - [Recommender API](https://console.cloud.google.com/apis/library/recommender.googleapis.com),
+  - [Service Usage API](https://pantheon.corp.google.com/apis/library/serviceusage.googleapis.com),
```
Comment (minorczyka) [inline README.md:78]: Please replace `pantheon.corp` with `console.cloud`, it's our internal name and link won't work for other people.

File: README.md [hunk 2]
```diff
@@ -60,6 +66,129 @@ npm run lint
...
+Now run 
+```
+gcloud app describe | grep defaultHostname
+```
+You will see the value of defaultHosthame (your future app address).
+
+Copy it, you’ll need it in the next step.
+
+### Create credentials
+ - Go to APIs & Services -> OAuth consent screen.
```
Comment (minorczyka) [inline README.md:117]: Can we add a link here?

File: README.md [hunk 3]
```diff
@@ -60,6 +66,129 @@ npm run lint
...
+ 
+ - Add scopes: 
+```
+.../auth/userinfo.email
+.../auth/cloud-platform
+```
+
+- Click `Save`, then `Back to dashboard`.
+
+- Go to Credentials-> Create Credentials -> OAuth client ID
```
Comment (minorczyka) [inline README.md:135]: Link here?

**Review by minorczyka — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ViktoriiaKovalova) [reply README.md]: yes. I deleted the images, because with the link it should be easy to get there now.

**Review by minorczyka — CHANGES_REQUESTED** [review]
If you decided to remove these two images from instructions, please remove also the files from `docs/`.

**Review by minorczyka — APPROVED** [review]

**Pull request merged**